t/11_hacluster.t

```diff
@@ -90,4 +90,30 @@ subtest '[script_output_retry_check] Diskless SBD scenario' => sub {
     set_var('USE_DISKLESS_SBD', undef);
 };
 
+subtest '[cluster_status_matches_regex]' => sub {
+    my $hacluster = Test::MockModule->new('hacluster', no_auto => 1);
```

Comment:
This line is not needed as you do not need to mock anything for this test

```
my $hacluster = Test::MockModule->new('hacluster', no_auto => 1);
```

Reply:
Does this test pass? Do you not need to mock `record_info` ?

Reply:
No it is no passing, throws an error about record info method, that's why I need to mock and use hacluster variable

Reply:
sorry, discard my comment. both `Test::MockModule->new` and `$hacluster->redefine(record_info ` are needed